docker/lite/build.sh

```diff
@@ -14,7 +14,9 @@ fi
 
 # Extract files from vitess/base image
 mkdir base
-sudo docker run -ti --rm -v $PWD/base:/base -u root $base_image bash -c 'cp -R /vt /base/'
+# Ignore permission errors. They occur for directories we do not care e.g. ".git".
+# (Copying them fails because they are owned by root and not $UID and have stricter permissions.)
+docker run -ti --rm -v $PWD/base:/base -u $UID $base_image bash -c '{ cp -R /vt /base/ 2> >(grep -v "Permission denied" >&2); }'
```

Comment:
I don't understand this syntax. Why do you use curly braces? What the added part mean? Should this be replaced with the following?
<code>
cp -R /vt /base/ 2>&1 | grep -v "Permission denied"
<code>